api/lib/source/cosmosV0-source.js

```diff
@@ -121,8 +121,7 @@ class CosmosV0API extends RESTDataSource {
         throw Error()
       }
       const response = await this.query(
-        `slashing/validators/${validatorConsensusPubKey}/signing_info`,
-        { cacheOptions: { ttl: 60 } }
```

Comment:
what was this cacheOption about? :eyes: not needed?

Reply:
never used